content/en/docs/ipv6.md

```diff
@@ -0,0 +1,63 @@
+---
+title: IPv6 Support
+slug: ipv6-support
+top_graphic: 1
+date: 2020-02-07
+lastmod: 2020-02-07
+---
+
+{{< lastmod >}}
+
+Let's Encrypt supports IPv6 both for accessing the ACME API using an ACME
```

Comment:
s/made by our validation authority/we make/.

Reasoning: The reader doesn't need to know the internals of how we implement Boulder. Also, our terminology "validation authority" is actually at odds with the standards documents, which use that term to describe OCSP and CRL endpoints.

Maybe another way to phrase "and for the DNS lookups and HTTP requests we make when validating your control of domains names."